rsmtool/notebooks/evaluation.ipynb

```diff
@@ -133,10 +151,13 @@
    "outputs": [],
    "source": [
     "markdown_strs = [\"The histogram and the table below show the distibution of \"\n",
-    "                 \"human scores and {} trimmed rounded machine scores \"\n",
+    "                 \"rounded human scores and {} trimmed rounded machine scores \"\n",
```

Comment:
Can we use some commas? `scaled trimmed rounded` -> `scaled, trimmed, and rounded`?